pkg/frame/decode.go

```diff
@@ -17,6 +17,8 @@ const (
 	FormatNV12 = "NV12"
 	// FormatYUY2 https://www.fourcc.org/pixel-format/yuv-yuy2/
 	FormatYUY2 = "YUY2"
+	// FormatYUYV https://www.fourcc.org/pixel-format/yuv-yuy2/
```

Comment:
This link isn't working for me. I think it's gone stale. Maybe figure our if _fourcc.org_ has moved to another domain, and if so where. Or you could paste another authoritative link about the YUYV format.

Reply:
Wow yeah great catch, I just changed them in a new commit

Reply:
Thank you!